tests/DemoFiles/Test_Global_Context.json

```diff
@@ -66,12 +162,17 @@
     }
   },
   "ASP_Comps": {
-    "sig_id": "certificate_id",
-    "hashfile_id": "appraise_id",
-    "attest_id": "appraise_id",
-    "attest1_id": "appraise_id",
-    "attest2_id": "appraise_id",
-    "appraise_id": "appraise_id",
-    "certificate_id": "appraise_id"
+    "attest": "magic_appr",
+    "certificate": "magic_appr",
+    "appraise": "magic_appr",
+    "hashevidence": "NULL",
+    "hashfile": "hashfile_appr",
+    "invary_get_measurement": "invary_get_measurement_appr",
+    "provision": "NULL",
+    "readfile": "readfile_appr",
+    "selinux_pol_dump": "selinux_pol_dump_appr",
+    "sig": "sig_appr",
+    "sig_tpm": "sig_tpm_appr",
+    "uptime": "NULL"
```

Comment:
We may need to modify some of the NULL.

Additionally, look into creating a common appraisal ASP for golden value checks (i.e. readfile, provision, hashfile, etc.)